bestiary/core/db.py

```diff
@@ -328,6 +332,79 @@ def delete_project(trxl, project):
                        target=str(op_args['id']))
 
 
+def link_parent_project(trxl, project, parent_project):
+    """Link a project object with another with a child-parent relation.
+
+    This method sets the `parent_project` field value from `project` with the
+    `Project` object from `parent_project`. If this field already had a value,
+    it will be overwritten. In case the project and the parent project are the
+    same, the method will raise an `InvalidParentError`.
+
+    :param trxl: TransactionsLog object from the method calling this one
+    :param project: Source project to be linked
+    :param parent_project: Parent project to be linked to the source project
+
+    :returns: the updated project object
+
+    :raises InvalidParentError: raised either when the given project and the
+        given parent project are equal
+    """
+    # Setting operation arguments before they are modified
+    op_args = {
+        'id': project.id,
+        'parent_id': parent_project.id
+    }
+
+    if project == parent_project:
+        raise InvalidParentError(msg='Project cannot be its own parent')
+
+    project.parent_project = parent_project
+
+    try:
+        project.save()
+    except django.db.utils.IntegrityError as exc:
+        _handle_integrity_error(Project, exc)
+
+    trxl.log_operation(op_type=Operation.OpType.LINK, entity_type='project',
+                       timestamp=datetime_utcnow(), args=op_args,
+                       target=str(op_args['id']))
+
+    return project
+
+
+def link_to_ecosystem(trxl, project, ecosystem):
```

Comment:
This method was added because the `move_project` API method allowed to change the ecosystem where a project belongs. Although this method is not used yet, we will need it in the future so I decided to keep it (it has unit tests too).

Reply:
If we're not going to use it, it is better to remove it. Probably you can linked to the comments of this PR (because if we rebase we lost the history).